test/cypress/integration/payment-links/delete_payment_links_spec.js

```diff
@@ -0,0 +1,52 @@
+const commonStubs = require('../../utils/common_stubs')
+const { getProductsStub, getProductByExternalIdStub, deleteProductStub } = require('../../utils/products_stubs')
+const userExternalId = 'a-user-id'
+const gatewayAccountId = 42
+const productExternalId = 'a-product-id'
+
+const buildPaymentLinkOpts = function buildPaymentLinkOpts (name, href, language) {
+  return {
+    external_id: productExternalId,
+    name: name,
+    language: language,
+    type: 'ADHOC',
+    links: [
+      {
+        'rel': 'friendly',
+        'method': 'GET',
+        'href': href
+      }
+    ]
+  }
+}
+
+const product = buildPaymentLinkOpts('Pay for a parking permit', 'example.com/my-service/pay-for-a-parking-permit', 'en')
+
+describe('Should delete payment link', () => {
+  beforeEach(() => {
+    cy.setEncryptedCookies(userExternalId, gatewayAccountId)
+    cy.task('setupStubs', [
+      commonStubs.getUserStub(userExternalId, [gatewayAccountId]),
+      commonStubs.getGatewayAccountStub(gatewayAccountId, 'test', 'worldpay'),
```

Comment:
i know this was copy+pasted but could we use destructuring for these imports too for consistency.